src/screens/TopicPageScreen/index.js

```diff
@@ -395,6 +395,7 @@ const TopicPageScreen = (props) => {
       onPressDownVote={(post) => setDownVote(post, index)}
       loading={loading}
       selfUserId={userId}
+      offset={offset}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new `offset` prop is being passed to the component, but it's not clear where this value is coming from. Ensure that `offset` is defined and its value is correctly calculated before passing it as a prop.
```diff
-       offset={offset}
+       offset={calculatedOffset} // ensure 'calculatedOffset' is defined and correctly calculated
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->